src/meta/processors/admin/BalancePlan.h

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef META_ADMIN_BALANCEPLAN_H_
+#define META_ADMIN_BALANCEPLAN_H_
+
+#include <gtest/gtest_prod.h>
+#include "kvstore/KVStore.h"
+#include "meta/processors/admin/BalanceTask.h"
+
+namespace nebula {
+namespace meta {
+
+class BalancePlan {
+    friend class Balancer;
+public:
+    enum class Status : uint8_t {
+        START              = 0x01,
```

Comment:
Maybe NOT_STARTED will be better? It confuses me when I review at the first time.